batect.yml

```diff
@@ -48,7 +41,7 @@ containers:
       - type: cache
         name: gradle-build-dir
         container: /code/build
-      - local: <{batect.project_directory}
+      - local: .
```

Comment:
The project directory ends up being the folder where the root batect file lives, which breaks all of these commands.